doc/man1/openssl-ca.podin

```diff
@@ -766,8 +766,7 @@ the database has to be kept in memory.
 
 This command really needs rewriting or the required functionality
```

Comment:
```suggestion
This command really needs rewriting.
```
This is the only take-away of the entire paragraph and as such it applies to all "apps", so maybe just delete the paragraph altogether.

Reply:
:). deleted the paragraph here and in the `ts` page too